src/gc-pages.c

```diff
@@ -21,6 +21,19 @@ extern "C" {
 
 static int block_pg_cnt = DEFAULT_BLOCK_PG_ALLOC;
 static size_t current_pg_count = 0;
+static int support_conservative_scans = 0;
```

Comment:
So this commit is the kind of change that will be very hard to maintain... It relies on a lot of the internal implementation details that can otherwise be changed without any issue.

Reply:
Okay, I think I can now get rid of this part entirely by implementing your other suggestion for finding references to pool allocations, as that would (I think) eliminate the need to zero pages entirely. I'll put the code in and get back to you ASAP with an update (including also the other requested changes).